GiniVision/Classes/Core/Gallery/GalleryManager.swift

```diff
@@ -45,12 +50,19 @@ final class GalleryManager: GalleryManagerProtocol {
         }
     }
     
-    func fetchImageData(from asset: Asset, completion: @escaping ((Data) -> Void)) {
-        cachingImageManager.requestImageData(for: asset.value, options: nil) { data, _, _, _ in
-            if let data = data {
-                completion(data)
-            }
+    func fetchImageData(from asset: Asset, isRemote: Bool, completion: @escaping ((Data?) -> Void)) {
```

Comment:
throw new FlagArgumentDetectedException("The flag argument isRemote signals that this method does more than one thing. Consider moving the two execution branches into separate methods.")

I think fetchRemoteImageData() is a suitable name for the isRemote: true case.

Reply:
Yes, make sense for this case. 👍